main.c

```diff
@@ -218,6 +231,49 @@ static const struct wl_keyboard_listener keyboard_listener = {
 	.modifiers = noop,
 };
 
+static void touch_handle_down(void *data, struct wl_touch *touch, uint32_t serial,
+                          uint32_t time, struct wl_surface *surface, int32_t id,
+                          wl_fixed_t x, wl_fixed_t y) {
+	struct slurp_seat *seat = data;
+	if (!seat->already_touched) {
+		seat->already_touched = true;
+		seat->current_output =
+			output_from_surface(seat->state, surface);
+		seat->current_touch_output = seat->current_output;
+		move_pointer(seat, x, y);
```

Comment:
Since these coordinates are used for touch too, could rename to `move_seat`.

Reply:
And done f2050fa6a7ca7c4cc72e122970ed00fdf21bc5bd.